modules: lvgl: support link mem pool to customer section

ZhuZhongjie opened the pull request:
Add a Kconfig LV_Z_MEMORY_POOL_CUSTOM_SECTION to support change the lvgl heap pool to a custom section ".lvgl_heap".

pdgendt commented:
@ZhuZhongjie Please do not close and open new pull requests, simply force-push to overwrite your changes.

faxe1008 commented on modules/lvgl/Kconfig.memory:
```diff
@@ -29,6 +29,15 @@ choice LV_Z_MEMORY_POOL
+	  This can be used by custom linker scripts to relocate the LVGL
+	  memory pool to a custom location, such as tightly coupled or
+	  external memory.
+
```
Please move this out of the choice.

faxe1008 commented on modules/lvgl/Kconfig.memory:
```diff
@@ -29,6 +29,15 @@ choice LV_Z_MEMORY_POOL
 		help
```
Also: check the commit message, typo `customer`

ZhuZhongjie committed:
modules: lvgl: support link mem pool to customer section

Add a Kconfig LV_Z_MEMORY_POOL_CUSTOM_SECTION to support change
the lvgl heap pool in custom section ".lvgl_heap".

Signed-off-by: Zhu Zhongjie <zhongjiezhu1@gmail.com>
